### PR TITLE
[12.0][ADD] stock_picking_product_label

### PR DESCRIPTION
**Pull request by NuriaXifre**
New module that allows printing labels of all products in a stock picking.

cc ~ @ForgeFlow

### CODE REVIEW

**Comment by NuriaXifre** [comment]
Runbot error is not related

**Review by JoanSForgeFlow — APPROVED** [review]
Functional review done!

File: stock_picking_product_label/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2019 Eficent <patrickraymondwilson@gmail.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Print Labels in Picking',
+    'summary': "Print Product Labels in a Stock Picking",
+    'author': "Eficent, Odoo Community Association (OCA)",
+    'website': "https://github.com/OCA/stock-logistics-reporting",
+    'category': 'Customer Relationship Management',
```
Comment (rousseldenis) [inline stock_picking_product_label/__manifest__.py]: Change this to 'Warehouse' as it will appear in the wrong category in OCA shop

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Little change

File: stock_picking_product_label/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2019 Eficent <patrickraymondwilson@gmail.com>
```
Comment (HviorForgeFlow) [inline stock_picking_product_label/__manifest__.py]:
```suggestion
# Copyright 2019 Eficent Business and IT Consulting Services S.L.
```

File: stock_picking_product_label/readme/CONTRIBUTORS.rst [hunk 3]
```diff
@@ -0,0 +1,4 @@
+* Adrià Gil <adria.gil@eficent.com>
+* Joan Sisquella <joan.sisquella@eficent.com>
+* Núria Martín <nuria.martin@eficent.com>
+
```
Comment (HviorForgeFlow) [inline stock_picking_product_label/readme/CONTRIBUTORS.rst]: Drop one line,
Last line must be empty.

File: stock_picking_product_label/readme/DESCRIPTION.rst [hunk 4]
```diff
@@ -0,0 +1,3 @@
+This module allows users to print labels of all done products in a stock picking.
+
+
```
Comment (HviorForgeFlow) [inline stock_picking_product_label/readme/DESCRIPTION.rst]: Drop lines

**Review by HviorForgeFlow — COMMENTED** [review]
Minor comments about styling. Code review LGTM

File: stock_picking_product_label/reports/report_print_labels.xml [hunk 5]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
```
Comment (ps-tubtim) [inline stock_picking_product_label/reports/report_print_labels.xml]: I think `<data>` and `</data>` are unnecessary in v.12

**Review by ps-tubtim — COMMENTED** [review]
Function test: OK. Minor comment about code.

File: stock_picking_product_label/reports/report_print_labels_template.xml [hunk 6]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
```
Comment (ps-tubtim) [inline stock_picking_product_label/reports/report_print_labels_template.xml]:
```suggestion
```

File: stock_picking_product_label/reports/report_print_labels_template.xml [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+                                    <t t-call="product.report_simple_barcode">
+                                        <t t-set="product" t-value="move_line.product_id"/>
+                                    </t>
+                                </div>
+                            </t>
+                          </t>
+                        </t>
+                </t>
+        </template>
+     </data>
```
Comment (ps-tubtim) [inline stock_picking_product_label/reports/report_print_labels_template.xml]:
```suggestion
```

**Review by ps-tubtim — COMMENTED** [review]
I think <data> and </data> are unnecessary in v.12

**Review by ps-tubtim — APPROVED** [review]

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by LoisRForgeFlow — APPROVED** [review]
Code review :+1:

**Comment by LoisRForgeFlow** [comment]
All reviews attended, thanks @NuriaMForgeFlow!

/ocabot merge

**Pull request merged**